pkg/metrics/controller_manager.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	collector          controllerCollector
+	metricsInitialized bool
+
+	// ControllerWorkerSum is a metric descriptor which counts the amount of workers per controller.
+	ControllerWorkerSum = prometheus.NewDesc("garden_cm_worker_total", "Count of running controller workers", []string{"controller"}, nil)
```

Comment:
From the looks of things this is a gauge metric (was it counter before?). In this case we should drop the `_total`. Maybe you should the comment to `Current amount of controller workers`?

Reply:
No, it was always a gauge, but missed to apply the naming convention correct. 
Let's call the metric `garden_cm_worker_amount`, ok?

Reply:
Okay

Reply:
Done